### DISCUSSION TITLE
heavy cpu loading for mfsmount

### DISCUSSION THREAD

**Discussion by mttimyim** [moosefs/moosefs#623] [General]
We are currently conducting performance tests and have observed high CPU usage for mfsmount. Could you please assist us with this issue? Thank you.

**Comment by chogata** [comment]
Can you post more details? Are you using the latest version of MooseFS? What OS are you using? What operations are you performing on your mount exactly? Can you post a screenshot from your GUI from Operations tab?

**Reply by mttimyim** [reply]
OS: Rocky Linux release 9.4
MooseFS version: 4.56.6-1

Run mdtest on client

    PID USER      PR  NI    VIRT    RES    SHR S  %CPU  %MEM     TIME+ COMMAND
  38417 root       1 -19  568700  77120   1024 S 115.9   1.0   1474:52 mfsmount
 353299 ohpc-te+  20   0  171072  16128  13056 S  11.6   0.2   0:01.80 mdtest
 353300 ohpc-te+  20   0  171072  16128  13056 S  11.0   0.2   0:01.83 mdtest
 353301 ohpc-te+  20   0  171072  16128  13056 S  11.0   0.2   0:01.89 mdtest
 353302 ohpc-te+  20   0  171072  16128  13056 S  10.3   0.2   0:01.81 mdtest

**Comment by chogata** [comment]
We tried to test it and we don't have such heavy usage. What options did you use with `mdtest` command?

In general, a lot of metadata operations mean a lot of network traffic between the master and the client - this can also affect CPU, depending on your hardware and network setup.
